Source/ARTRealtimeChannel.m

```diff
@@ -866,11 +875,14 @@ - (void)onPresence:(ARTProtocolMessage *)message {
         if (!presence.id) {
             presence.id = [NSString stringWithFormat:@"%@:%d", message.id, i];
         }
-
+      
         if ([self.presenceMap add:presence]) {
             [self broadcastPresence:presence];
         }
-
+        
+        // RTL15b
+        self.channelSerial = message.channelSerial;
```

Comment:
Again I think this should be the `ProtocolMessage.channelSerial`

Reply:
This should be fixed as part of da8a0ba52406e50c52ff9a0de235f799eb04f9bc